bedrock/sitemaps/models.py

```diff
@@ -0,0 +1,68 @@
+import json
+
+from django.conf import settings
+from django.db import models, transaction
+
+SITEMAPS_DATA = settings.SITEMAPS_PATH.joinpath('data')
+
+
+def load_sitemaps_data():
+    with SITEMAPS_DATA.joinpath('etags.json').open() as fh:
+        etags = json.load(fh)
+
+    with SITEMAPS_DATA.joinpath('sitemap.json').open() as fh:
+        sitemap = json.load(fh)
+
+    return sitemap, etags
+
+
+def get_sitemap_objs():
+    objs = []
+    sitemap, etags = load_sitemaps_data()
+    for url, locales in sitemap.items():
+        if not locales:
+            locales = [""]
```

Comment:
nit: style consistency `s/"/'/`until you install `black` in bedrock :D

Reply:
lol... true. I copied a lot of this code from the www-sitemap-generator repo, which is using black :)